res/styles.css

```diff
@@ -500,14 +494,29 @@ label {
 }
 
 
+/* Datepicker (SMH at this CSS) */
```

Comment:
This CSS is awful - it is way too specific and verbose, with loooong nested rules and even (shudder) `!important`. However that's what the plugin author used, so I guess that's what we're stuck with :(